syntax = "proto3";
option cc_enable_arenas = true;
package gears.proto.player;

// ported from ::CPLayerInfo::eHanded
// WARNING: this enum is used in persisted and should not
// have the names or values of it's elements changed
enum Handed {

    // Added because protobuf requires the first element to be zero.
    // If this value is encountered then something has likely gone wrong.
    UNKNOWN_HANDEDNESS = 0;

    // was eRightHanded=1
    RIGHT_HANDED = 1;

    // was eLeftHanded
    LEFT_HANDED = 2;

}
